gprofiler/perf.py

```diff
@@ -18,21 +20,35 @@
 logger = get_logger_adapter(__name__)
 
 
+@functools.lru_cache(maxsize=1)
+def perf_path() -> str:
+    return resource_path("perf")
```

Comment:
why not do the lru_cache on `resource_path` itself?

Reply:
Smart one